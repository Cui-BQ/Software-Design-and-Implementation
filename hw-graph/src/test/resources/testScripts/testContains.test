# Create graphs
CreateGraph graph

# Add 3 node to graph
AddNode graph n0
AddNode graph n1
AddNode graph n2

# Check contains
Contains graph n0
Contains graph n1
Contains graph n2