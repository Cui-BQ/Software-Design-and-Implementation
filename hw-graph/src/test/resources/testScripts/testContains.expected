# Create graphs
created graph graph

# Add 3 node to graph
added node n0 to graph
added node n1 to graph
added node n2 to graph

# Check contains
graph contains n0: true
graph contains n1: true
graph contains n2: true